### PR TITLE
commit for ticket 7

### PR DESCRIPTION
**Pull request by rose-phillips**

### CODE REVIEW

File: __tests__/INSERT INTO comments.sql
Comment (callumu21) [inline __tests__/INSERT INTO comments.sql]: I would make sure that this is .gitignored

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js]: review_id shouldn't be on the request, just username and body

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js]: Make sure to test against what you know the comment object should look like. You provided the body, so hopefully it comes back the same

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js]: GET is a different method to post. You want to check what happens when you POST to the parametric endpoint with a valid but non-existent review_id

File: __tests__/app.test.js
Comment (callumu21) [inline __tests__/app.test.js:236]: Just missing a few more tests here. What happens when username isn't on the request body? What happens if the request body has other, unnecessary keys?

File: models/games-model.js
Comment (callumu21) [inline models/games-model.js]: We want to return the whole comment here, not just the body.

**Review by callumu21 — CHANGES_REQUESTED** [review]
Just a few changes required before approval. See my comments, and let me know if anything isn't clear

**Comment by Geo-Beard** [comment]
Looks good - all amendments made :) feel free to merge and move on!

**Pull request merged**
